src/send/SendRedesign.tsx

```diff
@@ -0,0 +1,277 @@
+    {
+      optionNum: '4',
+      title: t('sendRedesign.getStarted.options.four.title'),
+      subtitle: t('sendRedesign.getStarted.options.four.subtitle'),
+    },
+  ]
+
+  return (
+    <View style={getStartedStyles.container} testID={'SelectRecipient/GetStarted'}>
+      <Text style={getStartedStyles.subtitle}>{t('sendRedesign.getStarted.subtitle')}</Text>
+      <Text style={getStartedStyles.title}>{t('sendRedesign.getStarted.title')}</Text>
+      {options.map((params) => renderOption(params))}
+    </View>
+  )
+}
+
+const getStartedStyles = StyleSheet.create({
+  container: {
+    flexDirection: 'column',
+    backgroundColor: colors.gray1,
+    padding: 24,
+    margin: 24,
+    borderRadius: 10,
+  },
+  subtitle: {
+    ...typeScale.labelXXSmall,
+    color: colors.gray3,
+  },
+  title: {
+    ...typeScale.labelMedium,
+    color: colors.gray5,
+    paddingBottom: 10,
+  },
+  optionWrapper: {
+    flexDirection: 'row',
+    paddingVertical: 10,
+  },
+  optionNum: {
+    borderWidth: 1,
+    borderColor: colors.gray2,
+  },
+  optionNumText: {
+    ...typeScale.labelXSmall,
+    color: colors.dark,
+  },
+  optionText: {
+    paddingLeft: 10,
+    flexDirection: 'column',
+  },
+  optionTitle: {
+    ...typeScale.labelSmall,
+    color: colors.gray5,
+    paddingBottom: 5,
+  },
+  optionSubtitle: {
+    ...typeScale.bodyXSmall,
+    color: colors.gray3,
+  },
+})
+
+function SendRedesign() {
+  const { t } = useTranslation()
+
+  const [searchQuery, setSearchQuery] = useState('')
+  const throttledSearch = throttle((searchInput: string) => {
+    setSearchQuery(searchInput)
+  }, SEARCH_THROTTLE_TIME)
+
+  const [shouldImportContacts, setShouldImportContacts] = useState(false)
+
+  const recentRecipients = useSelector((state) => state.send.recentRecipients)
+  const shouldShowGetStarted = !recentRecipients.length
+
+  const defaultCountryCode = useSelector(defaultCountryCodeSelector)
+
+  const { recipientVerificationStatus, recipient, setSelectedRecipient } =
+    useFetchRecipientVerificationStatus()
+
+  const dispatch = useDispatch()
+
+  useAsync(async () => {
+    if (shouldImportContacts) {
+      // TODO: Currently if a user does not have a phone number linked, this silently fails
+      const permissionGranted = await requestContactsPermission()
+      if (permissionGranted) {
+        dispatch(importContacts())
+      }
+      setShouldImportContacts(false)
+    }
+  }, [shouldImportContacts])
+
+  const onPressContacts = () => {
+    ValoraAnalytics.track(SendEvents.send_redesign_invite)
+    setShouldImportContacts(true)
+  }
+
+  const onPressQR = () => {
+    ValoraAnalytics.track(SendEvents.send_redesign_scan_qr)
+    navigate(Screens.QRNavigator, {
+      screen: Screens.QRScanner,
+    })
+  }
+
+  const buildSections = (): Section[] => {
+    const sections = [{ key: t('sendRedesign.recents'), data: recentRecipients }].filter(
+      (section) => section.data.length > 0
+    )
+
+    return sections
+  }
```

Comment:
```suggestion
  const sections: Section[] = []
  if (recentRecipients.data.length) {
    sections.push({ key: t('sendRedesign.recents'), data: recentRecipients })
  }
```
code golf. requires you to not call "buildSections" later.. but that doesn't appear to be necessary.